lib/mathjax-helper.coffee

```diff
@@ -18,7 +19,7 @@ module.exports =
       configureMathJax()
     script.type   = "text/javascript";
     try
-      # atom.packages.resolvePackagePath('mathjax-wrapper') doesnt work but
+      # atom.packages.resolvePackagePath('mathjax-wrapper') doesn't work but
```

Comment:
Thanks for all the grammar and spelling corrections, I am not good at English :D